bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -35,11 +34,11 @@ spec:
         - |
           #!/bin/bash
           set -euo pipefail
+          TIMESTAMP=`date -u "+%Y-%m-%d %H:%M:%S"`
```

Comment:
`$(foo)` is preferred to `` `foo` `` because backticks have weird quoting semantics and `$()` is saner. (Throughout the patch.)

But also, you're generating a timestamp once at the start of the function, and then reprinting that same identical timestamp in front of multiple different lines, which doesn't make much sense. You should either only print the timestamp once, at the start of the script, or else print a new timestamp each time